connectors/opa/README.md

```diff
@@ -5,7 +5,7 @@ For each of the tasks related to the opa connector, execute the corresponding co
 
 * Build the connector: `make build`
```

Comment:
You removed the build target

Reply:
Not for opa. The removed build target above was in katalog.